### PR TITLE
Feature/47 handle csv verification with pogues external variables

### PR DESCRIPTION
**Pull request by davdarras**

### CODE REVIEW

File: src/test/java/fr/insee/publicenemy/api/infrastructure/queen/dto/SurveyUnitSerializerTest.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                 }
+              },
+              "stateData":{
+                 "currentPage":"0",
+                 "date":900000000,
+                 "state":"INIT"
+              }
+           }
+        ]""".replaceAll("\\s+","")
+                ,jsonSurveyUnits.replaceAll("\\s+",""));
```
Comment (nsenave) [inline src/test/java/fr/insee/publicenemy/api/infrastructure/queen/dto/SurveyUnitSerializerTest.java:137]: neat trick!

**Review by nsenave — COMMENTED** [review]
comments mostly for more javadoc 🙃 + some details

File: src/test/java/fr/insee/publicenemy/api/infrastructure/queen/dto/SurveyUnitSerializerTest.java (hunk 1 above)
Comment (davdarras) [reply src/test/java/fr/insee/publicenemy/api/infrastructure/queen/dto/SurveyUnitSerializerTest.java:137]: too bad it replaces spaces in data too...

Comment (nsenave) [reply src/test/java/fr/insee/publicenemy/api/infrastructure/queen/dto/SurveyUnitSerializerTest.java:137]: There might be a library to test differences between jsons maybe?

**Pull request merged**